rcl/src/rcl/security_directory.c

```diff
@@ -23,7 +24,6 @@
 # pragma clang diagnostic push
 # pragma clang diagnostic ignored "-Wembedded-directive"
 #endif
-#include "tinydir/tinydir.h"
```

Comment:
I believe that the surrounding `__clang__` pragmas can be removed now that the `tinydir` include is gone.